netam/dxsm.py

```diff
@@ -43,8 +44,10 @@ def __init__(
         branch_lengths: torch.Tensor,
         multihit_model=None,
     ):
-        self.nt_parents = nt_parents
-        self.nt_children = nt_children
+        self.nt_parents = nt_parents.str.replace(RESERVED_TOKEN_REGEX, "N", regex=True)
```

Comment:
Comment just above here as a counterpoint to the comment below about unmodified versions of nt_parents.